controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2300 @@
+      - file_groupowner_backup_etc_shadow
+      - file_owner_backup_etc_shadow
+      - file_permissions_backup_etc_shadow
+
+  - id: 6.1.6
+    title: Ensure permissions on /etc/gshadow are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupowner_etc_gshadow
+      - file_owner_etc_gshadow
+      - file_permissions_etc_gshadow
+
+  - id: 6.1.7
+    title: Ensure permissions on /etc/gshadow- are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupowner_backup_etc_gshadow
+      - file_owner_backup_etc_gshadow
+      - file_permissions_backup_etc_gshadow
+
+  - id: 6.1.8
+    title: Ensure permissions on /etc/group are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupowner_etc_group
+      - file_owner_etc_group
+      - file_permissions_etc_group
+
+  - id: 6.1.9
+    title: Ensure permissions on /etc/group- are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupowner_backup_etc_group
+      - file_owner_backup_etc_group
+      - file_permissions_backup_etc_group
+
+  - id: 6.1.10
+    title: Ensure no world writable files exist (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_permissions_unauthorized_world_writable
+
+  - id: 6.1.11
+    title: Ensure no unowned files or directories exist (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - no_files_unowned_by_user
+
+  - id: 6.1.12
+    title: Ensure no ungrouped files or directories exist (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_permissions_ungroupowned
+
+  - id: 6.1.13
+    title: Audit SUID executables (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    rules:
+      - file_permissions_unauthorized_suid
+
+  - id: 6.1.14
+    title: Audit SGID executables (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    rules:
+      - file_permissions_unauthorized_sgid
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7197
+  - id: 6.2.1
+    title: Ensure password fields are not empty (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 6.2.2
+    title: Ensure no legacy "+" entries exist in /etc/passwd (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - no_legacy_plus_entries_etc_passwd
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7198
+  - id: 6.2.3
+    title: Ensure root PATH Integrity (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
```

Comment:
Looking at the OVALs for these two rules, I think it will satisfy the benchmark to use them.

- `accounts_root_path_dirs_no_write` checks for group/world writable files in the root $PATH
- `root_path_no_dot` checks for `.` in root's $PATH, but also checks for the other things that the benchmark mandates (empty dirs & trailing `:` characters)

I have added these.